modules/cli/src/main/java/org/apache/ignite/cli/builtins/cluster/InitClusterRequest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.cli.builtins.cluster;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/**
+ * A request to the Cluster Init REST API.
+ */
+public class InitClusterRequest {
+    @SuppressWarnings("unused")
```

Comment:
I think you don't need to suppress anything here, you can set up IDEA to ignore fields annotated with `@JsonProperty`

Reply:
I though similarly, but PMD has its own opinion :) It just fails the build at TC if it sees an unused field.